src/v/redpanda/admin/debug_bundle.cc

```diff
@@ -238,15 +259,58 @@ ss::future<std::unique_ptr<ss::http::reply>> admin_server::delete_debug_bundle(
       std::move(rep));
 }
 
+namespace {
+
+ss::future<debug_bundle::result<debug_bundle::job_id_t>>
+get_debug_bundle_job_id(
+  debug_bundle::service& service, const ss::sstring& filename) {
```

Comment:
These pass-by-reference's are fine because we know that the arguments outlive the function, right?

Reply:
> These pass-by-reference's are fine because we know that the arguments outlive the function, right?

Good spot.

Reply:
> These pass-by-reference's are fine because we know that the arguments outlive the function, right?

Good spot.

Reply:
Passed the result and made it synchronous